spec/lucky/verify_accepts_format_spec.cr

```diff
@@ -0,0 +1,78 @@
+require "../spec_helper"
+
+include ContextHelper
+
+private class ActionThatAcceptsHtml < Lucky::Action
+  accepted_formats [:html], default: :html
+
+  get "/test" do
+    plain_text "yay"
+  end
+
+  property clients_desired_format : Symbol = :foo
```

Comment:
Is this supposed to be `_clients_desired_format` from the HTTP::Server::Context?

Reply:
No this override the whole method. We're basically mocking it out but without a mocking library